weekly-updates/weekly-update.2015-06-19.md

```diff
@@ -0,0 +1,47 @@
+* [Should I use Node.js or io.js? And which version?](https://strongloop.com/strongblog/should-i-use-node-js-or-io-js-and-which-version/) article by Stronloop
+* [Deploying Multiple Node.js Apps with strong-pm.io](https://strongloop.com/strongblog/deploying-multiple-node-js-apps-with-strong-pm-io/) article by Strongloop
+* [How to Implement Node + React Isomorphic JavaScript & Why it Matters](https://strongloop.com/strongblog/node-js-react-isomorphic-javascript-why-it-matters/) article by Strongloop
+* [The smartest way to program smart things: Node.js](http://radar.oreilly.com/2015/06/the-smartest-way-to-program-smart-things-node-js.html) article on O'Reilly Radar by Leah Hunter
+* Blog post by Sebastian Seilund: [Node.js async in practice: When to use what?](http://www.sebastianseilund.com/nodejs-async-in-practice)
+* [Not Really Single Page Apps](http://dailyjs.com/2015/06/19/not-really-single-page-apps/) blog post on DailyJS
+* [Containerizing Node.js Apps with Docker and StrongLoop](https://strongloop.com/strongblog/containerizing-node-js-apps-with-docker-and-strongloop/) article by Strongloop
+* [Built in Node.js](http://builtinnode.com/): startups, apps, projects using Node
+* Interview in [Free Software Magazine](http://www.freesoftwaremagazine.com/articles/interview_mikeal_rogers_nodejs_fork_ended_giant_unifying_step_forward) about the merge and the foundation
+* [Why Node.js is hitting the big time in Enterprise Markets](http://apmblog.dynatrace.com/2015/04/09/node-js-is-hitting-the-big-time-in-enterprise-markets/) and [Keeping things simple with Node.js](http://apmblog.dynatrace.com/2015/06/02/keeping-things-simple-with-node-js/) articles by Daniel Khan
+* Blog post by Paul Sauve: [Comparing NodeJS IDEs (now with ST3 and WebStorm!)](https://paulb.gd/comparing-nodejs-ides/)
+* [Understanding npm](https://unpm.nodesource.com/) and [blog post](https://medium.com/@nodesource/npm-is-massive-2bdd9417591c) about it by Nodesource.
+* [Node.js High Availability](https://www.box.com/blog/node-js-high-availability-at-box/) article by Box
+* Deploying a Scalable Node.js Application with Docker by Nodevember. Video on [YouTube](https://www.youtube.com/watch?v=uhNpSWI8MTM)
+* [Top 10 Reasons To Use Node.js](http://blog.modulus.io/top-10-reasons-to-use-node) article by Modulus.io
+* [Why Node.js is Ideal for the Internet of Things](http://www.programmableweb.com/news/why-node.js-ideal-internet-things/analysis/2014/07/31) article by ProgrammableWeb
+* [Painless Microsoft SQL Server with Node.js and seriate](http://developer.leankit.com/painless-sql-server-with-nodejs-and-seriate/) article by Leankit
+* Blog post: [Golden rules for JS introduction to your EE project](http://blog.upwardsmotion.com/golden-rules-for-js-introduction-to-your-ee-project/)
+* [NodeSecurity](https://nodesecurity.io/) project
```

Comment:
NodeSecurity project has begun since 2014.  We need to introduce this site as a weekly update??